plugin/pkg/admission/storage/storageclass/setdefault/admission.go

```diff
@@ -141,8 +142,9 @@ func getDefaultClass(lister storagev1listers.StorageClassLister) (*storagev1.Sto
 		return nil, nil
 	}
 	if len(defaultClasses) > 1 {
-		klog.V(4).Infof("getDefaultClass %d defaults found", len(defaultClasses))
-		return nil, errors.NewInternalError(fmt.Errorf("%d default StorageClasses were found", len(defaultClasses)))
+		klog.V(4).Infof("getDefaultClass %d defaults found, picking one randomly", len(defaultClasses))
+		rand.Seed(time.Now().UnixNano())
+		return defaultClasses[rand.Intn(len(defaultClasses))], nil
```

Comment:
@thockin I've used rand here for starters, but is there a heuristic we have in mind that we can employ here. I'm open to suggestions, thanks.

Reply:
Tim should we not be consistent in a multimaster deployment?

Reply:
I think that sorting it alphabetically and returning the first one will make it